language/evm/move-to-yul/src/attributes.rs

```diff
@@ -14,7 +14,8 @@ const EVM_ARITH_ATTR: &str = "evm_arith";
 const PAYABLE_ATTR: &str = "payable";
 const RECEIVE_ATTR: &str = "receive";
 const RECEIVE_FALLBACK_ATTR: &str = "fallback";
-const TEST_ATTR: &str = "evm_test";
+const EVM_TEST_ATTR: &str = "evm_test";
+const TEST_ATTR: &str = "test";
```

Comment:
What are the semantics of `evm_test` and the difference between this and a normal `test`? Are these tests that should only be run for the EVM?

One worry I have about adding (or continuing support for) the `evm_test` attribute, is that this will need to be added to the compiler so that it can filter out test functions outside of test mode.

Reply:
Good question! IIRC the `evm_test` attribute was added to enable us to implement some basic evm-based tests, without having to wait for proper unit test support. I'm planning to kill it very soon -- it could what comes right after this PR.

@wrwg please correct me if I said anything wrong.

Reply:
Yes, it was just a temporary hack.